tests/framework/pipeline_helper.h

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright (c) 2015-2023 The Khronos Group Inc.
+ * Copyright (c) 2015-2023 Valve Corporation
+ * Copyright (c) 2015-2023 LunarG, Inc.
+ * Copyright (c) 2015-2023 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#pragma once
+
+#include "layer_validation_tests.h"
+
+// Helper class for tersely creating create pipeline tests
+//
+// Designed with minimal error checking to ensure easy error state creation
+// See OneshotTest for typical usage
+struct CreatePipelineHelper {
```

Comment:
nitpick FFTI

If you add a methods to a struct. It's no longer a struct. It's a class.

```suggestion
class CreatePipelineHelper {
```

Just make the necessary members `public`

Reply:
agree, it was a struct with `public` which was more strange, but agree, just make it a class with `public` this a crazy struct doing too much